sw/device/tock/README

```diff
@@ -0,0 +1,43 @@
+Tock OS
+=======
+
+Tock OS is a secure embedded operating system for microcontrollers
+
+Documentation available at:
+https://www.tockos.org/
+https://github.com/tock/tock
+
+Rationale
+=========
+Upstream OpenTitan implementation is not using DIFs, which means that the
+peripheral drivers cannot be directly implemented upstream. Therefore it is not
+possible to use upstream OpenTitan board and chip implementations.
+
+The intention is, where possible, to use upstream packages. At the moment the
+only excpetions are board and chip packages, which are implemented and
+maintained downstream.
+
+Copyright
+=========
+Any files taken directly from the upstream must keep their original copyright.
```

Comment:
I suggest replacing the two sentencies in this section with "Some files in this subdirectory are imported directly from upstream Tock, so do not include our standard license header. See COPYRIGHT for the original upstream copyright notice."

Reply:
Already applied Sams suggestion (probably whilst you were writing this comment).